src/containers/Category.js

```diff
@@ -11,14 +12,15 @@ export const Category = () => {
     city,
     category
   );
+  const businessList = businesses.sort((a, b) => (a.name > b.name ? 1 : -1));
 
-  const BusinessesList = businesses
-    .sort((a, b) => (a.name > b.name ? 1 : -1))
```

Comment:
This was here to sort business alphabetically, will that still be the case?

Reply:
It's still there in an earlier line - the diff doesn't quite reflect the changes I made